src/App.vue

```diff
@@ -692,9 +691,9 @@ export default class App extends Mixins(
   }
 
   /** Used to check if the filing is in a valid state for changes. */
-  private hasInvalidFilingState (filing: any): boolean {
-    const filingStatus = filing.header.status
-    return filingStatus !== FilingStatus.DRAFT
+  private hasValidFilingState (filing: any): boolean {
+    const filingStatus = filing?.header?.status
```

Comment:
Safety check. Also I updated this method's logic to match its comment and usage.